src/components/wizards/new-vm/NewVmWizard.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { NewVmWizard } from './NewVmWizard';
+import { Button } from 'patternfly-react';
+import { os } from './OperatingSystem';
+
+class NewVmWizardExample extends React.Component {
+  state = {
+    show: false
+  }
+
+  storageClasses = [
+    {
+      name: 'storage name 1',
+      class: 'storage class 1',
+      size: 10
+    },
+    {
+      name: 'storage name 2',
+      class: 'storage class 2',
+      size: 20
+    },
+    {
+      name: 'storage name 3',
+      class: 'storage class 3',
+      size: 30
+    }
+  ]
+
+  namespaces = [
+    {
+      name: 'namespace1'
+    },
+    {
+      name: 'namespace2'
+    }
+  ]
+
+  flavors = [
+    {
+      name: 'flavor1'
+    },
+    {
+      name: 'flavor2'
+    }
+  ]
+
+  workloadProfiles = [
+    {
+      name: 'workloadProfile1'
+    },
+    {
+      name: 'workloadProfile2'
+    }
+  ]
+
+  templates = [
+    {
+      name: 'template1'
+    },
+    {
+      name: 'template2'
+    }
+  ]
+
+  openWizard = () => {
+    this.setState({show: true});
+  }
+
+  closeWizard = () => {
+    this.setState({show: false});
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <Button onClick={this.openWizard}>Open New VM Wizard</Button>
+          {this.state.show ? <NewVmWizard
+            closeWizard={this.closeWizard}
```

Comment:
I think onClose would be more fitting

Reply:
I agree, in React world, `onSomething` is a standard convention for callback props.

Reply:
renamed to onHide